src/store/TimeperiodStore.js

```diff
@@ -19,13 +19,59 @@ export default class TimeperiodStore {
 
     @observable timeUnit = null;
     @observable interval = null;
+    @observable remain = "00:00";
 
     onContextReady = () => {
         const { timeUnit, interval } = this.context.stx.layout;
         this.timeUnit = getTimeUnit({ timeUnit, interval });
         this.interval = interval;
+
+        const stx = this.context.stx;
+
+
+
+        reaction(() => this.timeUnit , () => { this.showCandleCountdown(); });
+
+        const displayMilliseconds = (ms) => {
+            const totalSec = ms / 1000;
+            if (totalSec <= 0) { return null; }
+            const padNum = (n) => (`0${n}`).slice(-2);
+            const seconds = padNum(Math.trunc((totalSec) % 60));
+            const minutes = padNum(Math.trunc((totalSec/60) % 60));
+            let hours = Math.trunc((totalSec / 3600) % 24);
+            hours = hours ? `${hours}:` : '';
+            return `${hours}${minutes}:${seconds}`;
+        };
+
+        setInterval(() => {
```

Comment:
sorry I didn't notice this earlier - this function will be called when the context is loaded. This means that even when the countdown is disabled it will still be called.

Reply:
Done!